src/i965_decoder_utils.c

```diff
@@ -355,6 +354,9 @@ avc_get_first_mb_bit_offset_with_epb(
             i += 2, j++, n++;
     }
 
+    if (buf)
+        free(buf);
+
```

Comment:
free() can accept NULL pointer, so it is unnecessary to check the pointer against NULL.